### ISSUE TITLE
Delay capture of livestream (Ustream.tv)

### ISSUE THREAD

**Issue by serendipiteez** [chrippa/livestreamer#1248]
Is there a setting to only start the capture after a period of time? e.g. wait for 2 hours from now and start capturing then

**Comment by scottbernstein-zz** [comment]
Not really, although I would find that very useful.  It would probably have to be something like --delay-start SECONDS -- where you'd pass in the number of seconds to wait before requesting the stream.  You'd pass in 3600 to wait an hour, 7200 to wait 2 hours, etc.

If nobody else has already implemented, I might take a stab at coding this myself.  :)

**Comment by skulblakka** [comment]
You can also write a simple batch script that starts livestreamer after a specified time. On Windows you can use `timeout /t x` to wait x seconds.

**Comment by scottbernstein-zz** [comment]
Committed a change to do exactly this:
https://github.com/scottbernstein/livestreamer/commit/900c4d199f721fb57d1e534e37bff11377d1c853
